src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,136 @@
+- This image can be included in your Okteto manifest's `deploy` and `destroy` sections
+- External resources are deleted when your Okteto namespace is deleted
+- External resources are covered by Okteto's garbage collection process
+
+## Configuring the `external` section
+
+Let's take a look at the composition of the `external` section in the Okteto manifest starting with the high-level structure of this configuration block. There are some important details you need to know before using this feature.
+
+Here's an example of how to define your `external` resources in the Okteto manifest, which you can see in our [References section](/reference/manifest/#external-object-optional):
+
+```yaml
+external:
+  db:
+    notes: docs/database.md
+    icon: database
+    endpoints:
+    - name: db
+      url: https://localhost:3306
+  functions:
+    notes: docs/lambdas.md
+    icon: function
+    endpoints:
+    - name: data-aggregator
+      url: https://fake-id.lambda-url.us-east-1.on.aws.aggregator
+    - name: data-processor
+      url: https://fake-id.lambda-url.us-east-1.on.aws.processor
+```
+
+> You can also use the [dynamic endpoints](/reference/manifest/#use-dynamic-endpoints-for-an-external-url) feature here to provide more flexibility in the URLs.
+
+Every keyword that you see here is opinionated except for `db:` and `functions:`. Those two keywords, the names of the external resources being defined here, can be anything you want them to be. For example, if we want to call the `db:` resource `dundie_award:` you can do so. Similarly, you could name `functions:` something like `parkour:` and it will work just fine.
+
+Whatever name you use to label these resources will be what's shown in the Okteto UI. Choose wisely!
+
+## Defining Your External Resources
+
+As you configure your external resources in this section, keep in mind the following definitions:
+
+`notes:` is used to provide context in the form of a relative link to a markdown file. The contents are displayed in the Okteto UI (see below).
+
+<p align="center"><Image src="/docs/cloud/images/external_resource_notes.png" alt="External resources notes section" width="450" /></p>
+
+`icon:` specifies one of several predefined icons you can use when this resource is rendered in the Okteto UI. The current list of supported icons is:
+
+- container
+- dashboard
+- database
+- default
+- function
+- graph
+- storage
+
+<p align="center"><Image src="/docs/cloud/images/external_resource_list.png" alt="External resources listed in dev environment" width="450" /></p>
+
+`endpoints:` contain information on how to access the external resource using:
+
+- `name:` to name the endpoint you're defining
+- `url:` to provide the specific url for accessing the resource
+
+The name of the endpoint is used to inject or modify environment variables. In this example, one of those would be: `OKTETO_EXTERNAL_FUNCTIONS_ENDPOINTS_DATA_AGGREGATOR_URL`.
```

Comment:
Can we add a section on dynamic endpoints? It's something that everyone will need for non-trivial samples.

Reply:
You bet.

I'll take a stab at it and we can review 👍

Reply:
Latest commit [takes a pass](https://github.com/okteto/docs/pull/363/files#diff-b5fcc1ba2f90c8eced81c0c31ce915bb44abc500bc7e6f24c7a4a998216e4769R127) at this. Let me know what you think 😄